ui/apps/dashboard/src/queries/URQLProvider.tsx

```diff
@@ -48,6 +51,13 @@ export function URQLProvider({ children }: { children: React.ReactNode }) {
             if (isUnauthenticatedError(error)) {
               // Log to Sentry if it still fails after trying to refresh the token and retrying the operation.
               Sentry.captureException(error);
+              signOut(() => {
+                router.push(
+                  `${
+                    process.env.NEXT_PUBLIC_SIGN_IN_PATH || '/sign-in'
+                  }?error=unauthenticated` as Route
```

Comment:
non-blocking: It might be nice to make `"unauthenticated"` a const kind of like an enum from the other file to link the two expected and provided keys together. We could export it from the signup file.

Reply:
Good suggestion :). Added it. I couldn't export it from the sign-in file though, because Next.js restricts exports on that file. I added a file just beside it for it.